### ISSUE TITLE
Feedback from US General Services Administration?

### ISSUE THREAD

**Issue by aj-stein-nist** [mesur-io/draft-zundel-spice-glue-id#3]
Has anyone considered presenting this to staff in the [US General Services Administration](), particularly those may be directly or indirectly involved with the design and maintenance of sam.gov data store and supporting systems, perhaps the [Technology Transformation Services](https://www.gsa.gov/about-us/organization/federal-acquisition-service/technology-transformation-services/tts-solutions) or [Office of Government-Wide Policy (OGP)](https://www.gsa.gov/about-us/organization/office-of-governmentwide-policy)? The latter have experience building systems and supporting policy for such macro-scale solutions, so it may be worth reaching out the scope, content, and registry of identifiers are more complete and vetted.

**Comment by brentzundel** [comment]
I'm not sure this matches what the intentions are for glue ids. My view is that Glue is a syntactic and semantic wrapper for existing company identification schemes, combined with a registry for convenient lookup of those schemes.

**Comment by brentzundel** [comment]
I agree that this feedback should be sought once the scheme is more complete.

**Comment by pamelatech** [comment]
Is SAM.gov a good example to put into the spec?

**Comment by selfissued** [comment]
@aj-stein-nist, is there somebody at the GSA that you suggest we talk to?

**Comment by brentzundel** [comment]
Open for over a week with no additional feedback from OP. Closing.
